### ISSUE TITLE
Reduce cross compile timeout

### ISSUE THREAD

**Issue by bradleystachurski** [fedimint/fedimint#4488]
We increased the timeout in CI for the cross compile job since we're troubleshooting network issues. We should ensure we reduce back to 20 minutes.

Further context: https://github.com/fedimint/fedimint/pull/4476#issuecomment-1982263388

**Comment by elsirion** [comment]
Still relevant @bradleystachurski?

**Comment by bradleystachurski** [comment]
Yep! Thanks for checking https://github.com/fedimint/fedimint/pull/7301
